control_rate: 250
p_gain: [5,5,5,5,5,5]
d_gain: [0.0,0.0,0.0,0.0,0.0,0.0]
ff_gain: [0.0,0.0,0.0,0.0,0.0,0.0]
max_control_output: [0.3,0.3,0.3,0.3,0.3,0.3]
dead_zone: [0.0,0.0,0.0,0.0,0.0,0.0]
